packages/protocol/contracts/governance/BlockchainParameters.sol

```diff
@@ -75,6 +98,51 @@ contract BlockchainParameters is Ownable, Initializable {
     emit IntrinsicGasForAlternativeFeeCurrencySet(gas);
   }
 
+  /**
+   * @notice Sets the uptime lookback window.
+   * @param window New window.
+   */
+  function setUptimeLookbackWindow(uint256 window) public onlyOwner {
+    require(window >= 12 && window <= 720, "UptimeLookbackWindow must be within safe range");
+    require(
+      window <= getEpochSize() - 2,
```

Comment:
edge case: if `epochSize` is 1 or 0 this will underflow

Reply:
+1, use SafeMath